backend/src/main/java/ca/gov/dtsstn/vacman/api/data/entity/EventEntity.java

```diff
@@ -0,0 +1,79 @@
+package ca.gov.dtsstn.vacman.api.data.entity;
+
+import java.time.Instant;
+
+import org.immutables.builder.Builder;
+import org.springframework.core.style.ToStringCreator;
+
+import jakarta.annotation.Nullable;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
```

Comment:
```suggestion
@Entity("Event")
```

This is so if you reference the entity in any JPQL queries, you can refer to it as `Event` and not `EventEntity`.